src/homepage/index.js

```diff
@@ -19,6 +23,9 @@ const Home = ({ config }) => {
   const isProdBeta = isProd() && isBeta()
```

Comment:
`isProdBeta` is not being used with the new changes. The benefit of this flag was to display certain items in prod beta on the homepage. Can we re-add the flag to the new components and hide the prod beta links?

Previous link or content hidden on prod beta:

Filing doc link
```JS
{hideContent && (
  <li>
    <a href='/documentation/category/filing-instructions-guide'>
      HMDA Online FIG
      <NewIndicator />
    </a>
  </li>
)}
```

`Info for Data Users` column is completely hidden on prod beta.

Reply:
Fixed.